### DISCUSSION TITLE
Viewing topics in the legacy discourse server

### DISCUSSION THREAD

**Discussion by Goldthing** [ImageMagick/ImageMagick6#107] [General]
Is anyone else having trouble viewing old forum threads found by searching?
Whenever I get a search result (eg. http://www.imagemagick.org/discourse-server/viewtopic.php?t=8113) and click on it I get redirected back to https://legacy.imagemagick.org/discourse-server/ this is very frustrating as I cannot look anything up.

The problem appears to be with viewtopic.php as I can click on discussions on the forum and read them, it's just search that seems broken.

**Comment by urban-warrior** [comment]
The previous discourse server has been decommissioned.  Instead, we mirrored the pages and they currently reside @ https://legacy.imagemagick.org/discourse-server.    It will take a few weeks to a few months for the new links to make their way into the search engines indexes (e.g. Google).

**Comment by jelabarre59** [comment]
So how would we find a thread/topic in the meantime?  In particular I was trying top find the thread for "Inversion of Film Negative Discussion" (DDG linked it as https://www.imagemagick.org/discourse-server/viewtopic.php?t=32867) but even searching directly on the legacy server couldn't find it.

I expect I will have to create a new thread (can't find anything similar here).

**Comment by urban-warrior** [comment]
See https://legacy.imagemagick.org/discourse-server~/viewtopic882c.html.

**Reply by jelabarre59** [reply]
I get a 404 error for that link

**Reply by urban-warrior** [reply]
Fixed the link.  Try again.

**Reply by jelabarre59** [reply]
That link update worked.  Thanks.

Of course, the thread didn't help though.  But that's a topic for a different thread (already posted elsewhere).
